ansible/group_vars/go-waku.yml

```diff
@@ -48,3 +56,4 @@ open_ports_list:
   - { port: '{{ go_waku_websocket_port }}' }
   - { port: '{{ go_waku_libp2p_port }}' }
   - { port: '{{ go_waku_metrics_port }}', chain: 'VPN', ipset: 'metrics.hq' }
+  - { port: '{{ go_waku_disc_v5_port }}', protocol: 'udp' }
```

Comment:
We'll need to open the port also in Terraform.

Reply:
Done!

Reply:
Thanks.